### PR TITLE
WIP: gitlab-ci

### PR DESCRIPTION
**Pull request by Razzwan**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    refs:
+      - gitlab-ci
+    variables:
+      - $CI_EXTERNAL_PULL_REQUEST_IID
+
+Docker Build:
+  stage: Docker Build
+  script:
+    - npm i
+    - npm run test:ci
```
Comment (orenyodfat) [inline .gitlab-ci.yml:57]: lint ?

File: docker-compose.docker.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+  graph-node:
+    restart: always
+
+  ipfs:
+    restart: always
+
+  postgres:
+    restart: always
+
```
Comment (orenyodfat) [inline docker-compose.docker.yml:14]: whats this is for ?

File: package.json [hunk 3]
```diff
@@ -75,6 +75,9 @@
     "test": "wdio ./test/integration/wdio.conf.js",
     "test:integration": "wdio ./test/integration/wdio.conf.js --inspect",
     "test:integration:headless": "wdio ./test/integration/wdio-headless.conf.js",
+    "test:i1": "wdio ./test/integration/wdio-headless.conf.js --suite suite1",
+    "test:i2": "wdio ./test/integration/wdio-headless.conf.js --suite suite2",
+    "test:i3": "wdio ./test/integration/wdio-headless.conf.js --suite suite3",
```
Comment (orenyodfat) [inline package.json:80]: is this is needed now ?

File: test/integration/wdio.conf.ts [hunk 4]
```diff
@@ -10,6 +10,29 @@ const drivers = {
...
+      "./test/integration/proposal-genericSchemeEnsPublicResolver.e2e.ts",
+    ],
+    suite3: [
+      "./test/integration/proposal-genericSchemeEnsRegistry.e2e.ts",
+      "./test/integration/proposal-genericSchemeRegistryLookup.e2e.ts",
+      "./test/integration/proposal-genericSchemeStandardBounties.e2e.ts",
+      "./test/integration/proposal-schemeregistrar.e2e.ts",
+      "./test/integration/rewards.e2e.ts",
+    ],
+  },
```
Comment (orenyodfat) [inline test/integration/wdio.conf.ts:35]: is this is needed now ?

File: docker-compose.docker.yml (hunk 2 above)
Comment (Razzwan) [reply docker-compose.docker.yml:14]: This is for keeping services working. It is from [this docs](https://docs.docker.com/compose/production/#modify-your-compose-file-for-production)

File: package.json (hunk 3 above)
Comment (Razzwan) [reply package.json:80]: OK, removing

File: README.md [hunk 5]
```diff
@@ -2,6 +2,10 @@
 
 [![Build Status](https://travis-ci.org/daostack/alchemy.svg?branch=dev)](https://travis-ci.org/daostack/alchemy)
 
+[![pipeline status](https://gitlab.com/daostack/alchemy/badges/dev/pipeline.svg)](https://gitlab.com/daostack/alchemy/-/commits/dev)
+
+[![coverage report](https://gitlab.com/daostack/alchemy/badges/dev/coverage.svg)](https://gitlab.com/daostack/alchemy/-/commits/dev)
+
```
Comment (orenyodfat) [inline README.md:8]: please remove this one

File: package.json (hunk 3 above)
Comment (orenyodfat) [reply package.json:80]: please remove it..

Comment (Razzwan) [reply package.json]: Done

File: test/integration/wdio.conf.ts (hunk 4 above)
Comment (Razzwan) [reply test/integration/wdio.conf.ts]: Done

Comment (Razzwan) [reply test/integration/wdio.conf.ts]: Removed, but actually it is usefull for local testing. To add some specific test to suite in order to be able to run just only this one test separately

**Review by orenyodfat — APPROVED** [review]

**Pull request merged**